lesson04/assignment/css/style.scss

```diff
@@ -0,0 +1,52 @@
+
+// insperation source code https://thoughtbot.com/blog/transitions-and-transforms
+
+@mixin flexVar($flexType: center) {
+  display: flex;
+  justify-content: $flexType;
+}
+
+h1 {
+  margin-top: 10%;
+  text-align: center;
+  font-size: 800%;
+}
+
+// Nesting
+.image-box {
+  @include flexVar;
+  align-items: center;
+  flex-direction: column;
+  width: 15%;
+  margin: 1%;
+
+  img {
+    width: 100%;
+    border-radius: 50%;
+  }
+
+  h2 {
```

Comment:
I would avoid setting font sizes as a percentage.  Try using rem units instead.  Rems are based on your base font size, and normally 1rem = 16px unless you adjust your base font size.

Also line height: 0 is causing the names to overlap.  A line-height of 1 or 1.2 might be more appropriate.